### DISCUSSION TITLE
v0.0.11

### DISCUSSION THREAD

**Discussion by ObuchiYuki** [ObuchiYuki/DevToysMac#117] [Announcements]
# Install
You can use it by downloading `DevToys.app.zip`, extracting it, and opening it.

or

run `brew upgrade devtoys`

# New Features
- Add QR Code reader

<hr /><em>This discussion was created from the release <a href='https://github.com/ObuchiYuki/DevToysMac/releases/tag/0.0.11'>v0.0.11</a>.</em>

**Comment by rookievips** [comment]
can't upgrade from homebrew

**Reply by Al3ski** [reply]
Fails to start on Macbook Pro M1
